secrets/values.yaml

```diff
@@ -0,0 +1,90 @@
+# secret main file
+
+# global values
+global:
+  auth: "oidc"
+  authIdToken: "FIXME"
+  authVO: "Rubin.production"
+
+  activemq:
```

Comment:
Better to move activemq and idds from global to each component section. Confusing to have
```
global:
   idds:
   activemq:
```
and
```
idds:
msgsvc:
```

Reply:
Actually I suggest that you get rid of "global" section since that is rather pointless. All info specified in this yaml are more or less global. E.g. password of panda-database is shared by panda/jedi/pandamon.
You just put
```
rucio:
oidc:
idds:
panda:
...
```
instead of
```
global:
  rucio:
  oidc:
  idds:

panda:
idds:
```

Reply:
updated.